config/webpack.common.js

```diff
@@ -189,7 +189,7 @@ module.exports = function (options) {
             }, {
               loader: 'css-loader',
               options: {
-                minimize: isProd,
+                minimize: true,
```

Comment:
The point of having a variable here is so that during development you can have the styles un-minimized.